packages/kv-store/src/Redis/RedisInitializer.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Tempest\KeyValue\Redis;
+
+use Predis;
+use Tempest\Container\Container;
+use Tempest\Container\DynamicInitializer;
+use Tempest\Container\Singleton;
+use Tempest\EventBus\EventBus;
+use Tempest\KeyValue\Redis\Config\RedisConfig;
+use Tempest\Reflection\ClassReflector;
+use UnitEnum;
+
+final class RedisInitializer implements DynamicInitializer
```

Comment:
`RedisClientInitializer`?

Reply:
See previous response, will rename if you really don't want a beautiful, nice, clean `Redis` interface 😌